backend/tests/integration/mailing/test_cli.py

```diff
@@ -4,58 +4,62 @@
 #
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
+import json
+from datetime import datetime
 
 import mock
-import pytest
 from freezegun import freeze_time
 from helpers.providers.faker import faker
 
 from inspirehep.mailing.cli import mailing
 
 
-@pytest.mark.vcr()
-def test_send_weekly_jobs(
-    app_cli_runner, base_app, db, es_clear, create_jobs, vcr_cassette
-):
-    result = app_cli_runner.invoke(mailing, ["send_weekly_jobs"])
+@freeze_time(datetime(2019, 9, 17, 6, 0, 0))
+def test_update_weekly_jobs(app_cli_runner, base_app, db, es_clear, create_jobs, redis):
+    config = {
+        "WEEKLY_JOBS_EMAIL_REDIS_KEY": "MAILTRAIN_KEY",
+        "WEEKLY_JOBS_EMAIL_TITLE": "Weekly jobs",
+    }
+    with mock.patch.dict(base_app.config, config):
+        result = app_cli_runner.invoke(mailing, ["update_weekly_jobs"])
     assert result.exit_code == 0
-    assert "Campaign sent" in result.output
-    assert vcr_cassette.all_played
-
-
-def test_send_weekly_jobs_with_no_jobs(app_cli_runner, db, es_clear):
-    result = app_cli_runner.invoke(mailing, ["send_weekly_jobs"])
+    assert "Campaign updated" in result.output
+
+    expected_redis_content = [
+        str(datetime(2019, 9, 17, 6, 0, 0).timestamp()),
+        "Weekly jobs",
+        '<!doctype html>\n<html xmlns="http://www.w3.org/1999/xhtml"',
+    ]
+    expected_keys = ["timestamp", "title", "html"]
+    redis_content = redis.hmget(config["WEEKLY_JOBS_EMAIL_REDIS_KEY"], expected_keys)
+    assert expected_redis_content[0] == redis_content[0]
+    assert expected_redis_content[1] == redis_content[1]
+    assert redis_content[2].startswith(expected_redis_content[2])
```

Comment:
I feel like this isn't testing if correct html template is used properly or not.